source/common/upstream/upstream_impl.cc

```diff
@@ -329,7 +329,21 @@ ClusterInfoImpl::ResourceManagers::load(const envoy::api::v2::Cluster& config,
   uint64_t max_pending_requests = 1024;
   uint64_t max_requests = 1024;
   uint64_t max_retries = 3;
-  const std::string runtime_prefix = fmt::format("circuit_breakers.{}.{}.", cluster_name, priority);
+
+  std::string priority_name;
+  switch (priority) {
+  case envoy::api::v2::RoutingPriority::DEFAULT:
+    priority_name = "default";
```

Comment:
There is a way to get the enum value as a string, https://stackoverflow.com/questions/32161409/how-to-get-protobuf-enum-as-string. I could have sworn I used it somewhere, but can't remember where. It doesn't matter for this simple example, but just FYI in case you have to do a large switch statement like this in the future.